mac-setup-elevated.sh

```diff
@@ -79,7 +79,7 @@ fi
 # Add github to known_hosts (one less prompt when QAing script)
 mkdir -p ~/.ssh
 grep -q github.com ~/.ssh/known_hosts 2>/dev/null || \
-    echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==" \
+    echo "github.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQCj7ndNxQowgcQnjshcLrqPEiiphnt+VTTvDP6mHBL9j1aNUkY4Ue1gvwnGLVlOhGeYrnZaMgRK6+PKCUXaDbC7qtbW8gIkhL7aGCsOr/C56SJMy/BCZfxd1nWzAOxSDPgVsmerOBYfNqltV9/hWCqBywINIR+5dIg6JTJ72pcEpEjcYgXkE2YEFXV1JHnsKgbLWNlhScqb2UmyRkQyytRLtL+38TGxkxCflmO+5Z8CSSNY7GidjMIZ7Q4zMjA2n1nGrlTDkzwDCsw+wqFPGQA179cnfGWOWRVruj16z6XyvxvjJwbz0wQZ75XK5tKSb7FNyeIEs4TT4jk+S4dhPeAUC5y+bDYirYgM4GC7uEnztnZyaVWQ7B381AK4Qdrwt51ZqExKbQpTUNn+EjqoTwvqNj4kqx5QUCI0ThS/YkOxJCXmPUWZbhjpCg56i+2aB6CmK2JGhn57K5mj0MNdBXA4/WnwH6XoPWJzK5Nyu2zB3nAZp+S5hpQs+p1vN1/wsjk=" \
```

Comment:
looks good.  it matches what we get here 
```
curl -L https://api.github.com/meta | jq -r '.ssh_keys | .[]' | sed -e 's/^/github.com /'
```

I wonder if that's what we want to do to automate this...

Reply:
My understanding is that the key rotation was an exceptional, unexpected event, and therefore isn't really worth automating.

Reply:
That's a handy command, though. Thanks for double checking!

Reply:
sounds good!

Reply:
I think we should do this the "right" way, which is to run:
```
ssh-keyscan -H github.com >> .ssh/known_hosts
```

Reply:
(Also, why is this in a mac-specific file?  Shouldn't we do this for linux as well?)

Reply:
> I think we should do this the "right" way, which is to run:
Yeah `curl -L https://api.github.com/meta` gives you the key details in a nice JSON format that `ssh-keyscan` also gives us.   Im impartial to which one we use.  I see less downsides than having to update dotfiles manually when someone runs into the issue.

> (Also, why is this in a mac-specific file? Shouldn't we do this for linux as well?)
You are right! This also affects linux.